composeApp/src/commonMain/kotlin/band/effective/office/elevator/ui/authorization/AuthorizationComponent.kt

```diff
@@ -15,16 +17,19 @@ import com.arkivanov.decompose.value.Value
 import com.arkivanov.essenty.parcelable.Parcelable
 import com.arkivanov.essenty.parcelable.Parcelize
 import com.arkivanov.mvikotlin.core.store.StoreFactory
+import org.koin.core.component.KoinComponent
+import org.koin.core.component.inject
 
 class AuthorizationComponent(
     componentContext: ComponentContext,
     private val storeFactory: StoreFactory,
     private val openContentFlow: () -> Unit
 ) :
-    ComponentContext by componentContext {
+    ComponentContext by componentContext, KoinComponent {
 
     private val validator: Validator = Validator()
     private val navigation = StackNavigation<AuthorizationComponent.Config>()
+    private val signInClient: GoogleSignIn by inject()
```

Comment:
why do you need `GoogleSignIn` in `AuthorizationComponent`? You can inject that In `AuthorizationGoogle`